agentai/testing.py

```diff
@@ -0,0 +1,35 @@
+from api import chat_complete
```

Comment:
Rewrite this using `pytest` please? 

Add `pytest` as a dev dependency to pyproject (refer Poetry docs!)